### ISSUE TITLE
az acr build from gitlabrunner

### ISSUE THREAD

**Issue by garzontaleroj** [Azure/azure-cli#18812]
## Describe the bug

**Command Name**
`az acr build`

**Errors:**
```
The command failed with an unexpected error. Here is the traceback:
'NoneType' object is not callable
Traceback (most recent call last):
  File "/home/gitlab-runner/lib/azure-cli/lib/python3.7/site-packages/knack/cli.py", line 231, in invoke
    cmd_result = self.invocation.execute(args)
  File "/home/gitlab-runner/lib/azure-cli/lib/python3.7/site-packages/azure/cli/core/commands/__init__.py", line 657, in execute
    raise ex
  File "/home/gitlab-runner/lib/azure-cli/lib/python3.7/site-packages/azure/cli/core/commands/__init__.py", line 720, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
  File "/home/gitlab-runner/lib/azure-cli/lib/python3.7/site-packages/azure/cli/core/commands/__init__.py", line 691, in _run_job
    result = cmd_copy(params)
  File "/home/gitlab-runner/lib/azure-cli/lib/python3.7/site-packages/azure/cli/core/commands/__init__.py", line 328, in __call__
    return self.handler(*args, **kwargs)
  File "/home/gitlab-runner/lib/azure-cli/lib/python3.7/site-packages/azure/cli/core/commands/command_operation.py", line 121, in handler
    return op(**command_args)
  File "/home/gitlab-runner/lib/azure-cli/lib/python3.7/site-packages/azure/cli/command_modules/acr/build.py", line 109, in acr_build
    variant=platform_variant
TypeError: 'NoneType' object is not callable

```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az acr build --file {} --registry {} --image {} {}`

## Environment Summary
```
Linux-5.8.0-59-generic-x86_64-with-debian-bullseye-sid
Python 3.7.10
Installer: 

azure-cli 2.26.0

```

**Comment by yonzhan** [comment]
route to service team

**Comment by northtyphoon** [comment]
The issue has been fixed since 2.26.1
